frontend/src/components/navigation/MenuItems.js

```diff
@@ -0,0 +1,34 @@
+export const MenuItems = [
+  {
+      title: 'Alumni',
+      url: '#',
+      cName: 'nav-links'
+  },
+  {
+      title: 'Events',
+      url: '#',
+      cName: 'nav-links'
+  },
+
+  {
+      title: 'Projects',
+      url: '#',
+      cName: 'nav-links'
+  },
+  {
+    title: 'Stories',
+    url: '#',
+    cName: 'nav-links'
+  },
+  {
+    title: 'Team',
+    url: '#',
+    cName: 'nav-links'
+  },
+  /*{
+      title: 'Sign up',
+      url: '#',
+      cName: 'nav-links-mobile'
+  }*/
+
+]
```

Comment:
nit: Add an ending newline to the file. This is just a style preference and we'll keep this as the default moving forward.